app/presenters/menu/item.rb

```diff
@@ -10,7 +10,7 @@ def name
     end
 
     def visible?
-      ApplicationHelper.role_allows_intern(rbac_feature)
+      ApplicationHelper.role_allows(rbac_feature)
```

Comment:
On the plus side, I'm glad that @Fryguy and @chessbyte thought the same thing as Chris and I thought as we started moving role_allows from App Helper and role_allows?/role_allows_any? from User.

Perhaps we can deprecate the App Helper role_allows so we can move to role_allows? in all the callers.